matchmake-extension/auto_matchmake_with_param_postpone.go

```diff
@@ -37,10 +64,19 @@ func (commonProtocol *CommonProtocol) autoMatchmakeWithParamPostpone(err error,
 		session = sessions[0]
 	}
 
-	errCode := common_globals.AddPlayersToSession(session, []uint32{connection.ID}, connection, "")
```

Comment:
Try something like
```go
var errCode nex.Error // I think thats what the type is, assuming based off the diff

if useAdditionalParticipants {
  errCode = common_globals.MovePlayersToSession(session, additionalParticipants, connection, autoMatchmakeParam.JoinMessage.Value)
} else {
  errCode = common_globals.AddPlayersToSession(session, []uint32{connection.ID}, connection, autoMatchmakeParam.JoinMessage.Value)
}
if errCode != nil {
    common_globals.Logger.Error(errCode.Error())
    return nil, errCode
}
```
to prevent the repetitiveness